src/hub.js

```diff
@@ -255,7 +250,10 @@ const onReady = async () => {
   const environmentRoot = document.querySelector("#environment-root");
 
   const initialEnvironmentEl = document.createElement("a-entity");
-  initialEnvironmentEl.addEventListener("bundleloaded", () => uiRoot.setState({ initialEnvironmentLoaded: true }));
+  initialEnvironmentEl.addEventListener("bundleloaded", () => {
+    uiRoot.setState({ initialEnvironmentLoaded: true });
+    setTimeout(() => scene.renderer.animate(null), 100);
```

Comment:
Why is this timeout needed? Can it be 0?

Reply:
Yeah, you're right, changed it to 0 timeout and added a comment. Just need to wait one tick to actually allow the environment to render.